pkgs/development/tools/build-managers/buildbot/plugins.nix

```diff
@@ -25,14 +25,14 @@ in {
   www = pythonPackages.buildPythonPackage rec {
     name = "${pname}-${version}";
     pname = "buildbot_www";
-    version = "0.9.7";
+    version = "0.9.9.post1";
 
     # NOTE: wheel is used due to buildbot circular dependency
```

Comment:
@nand0p I was talking about the only comment visible, which I suppose is not part of this PR, but still. 
` # NOTE: wheel is used due to buildbot circular dependency`